rest/method_get.go

```diff
@@ -58,6 +59,20 @@ func listGet(ctx context.Context, r *http.Request, route *RouteMatch) (status in
 		e = NewError(err)
 		return e.Code, nil, e
 	}
+	// Now we can set Total.
+	list.Total = len(list.Items)
```

Comment:
you are right. I was on the wrong path here. But why can't we just say:
```go
total, err := c.Count()
if err != nil {
	return nil, err
}
...
list := &resource.ItemList{Total: total, Items: []*resource.Item{}}
...
```

to get the total items at least in rest-layer-mongo? Is it because of performance considerations?